src/sage/geometry/cone.py

```diff
@@ -6364,9 +6364,8 @@ def random_cone(lattice=None, min_ambient_dim=0, max_ambient_dim=None,
 
     Or one that isn't strictly convex::
 
-        sage: K = random_cone(min_ambient_dim=5, min_rays=2,
-        ....:                 strictly_convex=False)
-        sage: K.is_strictly_convex()
+        sage: K = random_cone(min_ambient_dim=5, min_rays=2, strictly_convex=False) # skip_conda random timeouts
+        sage: K.is_strictly_convex() # skip_conda random timeouts
```

Comment:
This one stands out to me because I wrote `random_cone()`. I don't think this example should time out. All we have to do here is generate a subspace (or even half-space) of dimension 5 or more, which is pretty easy to do by generating a bunch of random vectors and adding them to the list of generators one at a time. You have to be very unlucky for it to not be "instantaneous."

Reply:
See eg https://github.com/sagemath/sage/actions/runs/6827937663/job/18571052628#step:11:6663